eth-contracts/test/audiusToken.test.js

```diff
@@ -7,10 +7,10 @@ contract('AudiusToken', async (accounts) => {
   let registry, token, governance
 
   // expected initial token values
-  const NAME = "Audius"
-  const SYMBOL = "AUDIO"
-  const DECIMALS = 18  // standard - imitates relationship between Ether and Wei
-  const INITIAL_SUPPLY = Math.pow(10,27) // 10^27 = 1 billion tokens, 18 decimal places
+  const NAME = 'Audius'
```

Comment:
Just a minor nitpick for the future:
With this many formatting changes, it's a bit hard to find the code/logic changes. Maybe either
* Add a bit to the PR to tell reviewers which commits to look at specifically
* Use a feature branch and do two separate PRs: one for the code changes, one for the linting updates. Then merge the feature branch once both sub-PRs have been merged.
* Add comments to the PR specifically in the areas where things need review.

Reply:
Yeah 100%. this was horrible practice - no denying that- but in the vein of being able to get something since I've been sitting on various attempts of this fix for like 2 months, I did this. I think what I would have had to do would be to turn off all my IDE lint settings, make the changes, then re-add them because the existing code didn't even pass the old lint in the repo. 

The only changes though were the things called out in the PR description. 

Thanks for reviewing! Sorry for the headache